core/src/main/java/com/scalar/db/storage/cosmos/CosmosConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.scalar.db.storage.cosmos;
+
+import com.google.common.base.Strings;
+import com.scalar.db.config.DatabaseConfig;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+import java.util.Properties;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+@SuppressFBWarnings("JCIP_FIELD_ISNT_FINAL_IN_IMMUTABLE_CLASS")
+public class CosmosConfig extends DatabaseConfig {
+
+  public static final String PREFIX = DatabaseConfig.PREFIX + "cosmos.";
```

Comment:
I'm wondering if the metadata configuration should be done per database implementation.
Is it separated to make different configurations for each database when using multi-storage?
(e.g., cosmos uses `cosmos_metadata` and dynamo uses `dynamo_metadata`?)

Reply:
Yes, the metadata configuration is storage implementation specific. Actually, Cassandra doesn't have the metadata table. And maybe even when using multi-storage, I think we should have separate configurations because users might want to have different metadata names for each storage.